functions/chain.links.pwd.fish

```diff
@@ -6,7 +6,7 @@ function chain.links.pwd
   set -l path "$PWD"
 
   # Replace either HOME with ~ or abbreviate project root.
-  if vcs.present
+  if vcs.refresh && vcs.present
```

Comment:
One big issue here is that `vcs.refresh` is quite slow to be called for each and every prompt. Perhaps a better approach could be the following:
```suggestion
  if vcs.present
    functions -q vcs.root || vcs.refresh
```
Would you please confirm this works for you?

Reply:
Yeah this works just fine! Thanks for the suggestion